src/routes/profilePage/musicCreator.jsx

```diff
@@ -18,6 +21,16 @@ export default function MusicCreator() {
         followerCount: 69
     };
 
+    // https://github.com/reactivestack/cookies/tree/master/packages/react-cookie/#getting-started
+    const [cookies] = useCookies(['username']); //username cookie, used for login
+    const navigate = useNavigate();
+
+    useEffect(() => { 
+        if (cookies.username != "musician") //user is not a music creator
```

Comment:
in the future, we still have to call validateLogin on the cookies since they could be set by the user.
maybe we should export that function from navbar or make a new file called security